src/com/lftechnology/java/training/niraj/leaguetable/LeagueInfo.java

```diff
@@ -0,0 +1,210 @@
+/**
+ * 
+ */
+package com.lftechnology.java.training.niraj.leaguetable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * 
+ * Helper class containing methods and properties that describes league table
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public class LeagueInfo {
+
+	private int wins = 0;
+	private int loss = 0;
+	private int draw = 0;
+	private int points;
+	private int goalFor = 0;
+	private int goalAgainst = 0;
+	private String team;
+	private String result;
+	private String[] resultList;
+
+	public LeagueInfo(String result, String team) {
+		setResult(result);
+		setTeam(team);
+		setResultList(parseResults());
+	}
+
+	public int getWins() {
+		return wins;
+	}
+
+	public int getLoss() {
+		return loss;
+	}
+
+	public int getDraw() {
+		return draw;
+	}
+
+	public int getPoints() {
+		return points;
+	}
+
+	public int getGoalFor() {
+		return goalFor;
+	}
+
+	public int getGoalAgainst() {
+		return goalAgainst;
+	}
+
+	public String getTeam() {
+		return team;
+	}
+
+	public void setTeam(String team) {
+		this.team = team;
+	}
+
+	public String getResult() {
+		return result;
+	}
+
+	public void setResult(String result) {
+		this.result = result;
+	}
+
+	public String[] getResultList() {
+		return resultList;
+	}
+
+	public void setResultList(String[] resultList) {
+		this.resultList = resultList;
+	}
+
+	/**
+	 * Parse the list of result in a string to an array
+	 * 
+	 * @return {@link String[]}
+	 * @author Niraj Rajbhandari<nirajrajbhandari@lftechnology.com>
+	 */
+	private String[] parseResults() {
+		String[] resultList = this.result.split(",");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "resultList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "resultList" which hides the field declared at line 26. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)